cucumber/policy/features/support/resource_helpers.rb

```diff
@@ -0,0 +1,34 @@
+module ResourceHelpers
+  include FullId
+
+  attr_reader :result
+
+  def get_resource(kind, id = nil)
+    client(uri('resources', kind, id)).get(header)
+  end
+
+  def get_privilaged_roles(kind, id, privilege)
+    client(uri('resources', kind, id)).get(
+      Authorization: create_token_header, params: { permitted_roles: true, privilege: privilege }
+    )
+  end
+
+  def add_secret(token, kind, id, value)
+    client(uri('secrets', kind, id)).post(value, header(token))
+  end
+
+  def get_secret(token, kind, id)
+    client(uri('secrets', kind, id)).get(header(token))
+  end
+
+  def get_roles(kind, id)
+    client(uri('roles', kind, id)).get(header)
+  end
+
+  def client url
+    RestClient::Resource.new(url, 'Content-Type' => 'application/json')
+  end
```

Comment:
What's the reason we use `client` in some of the helpers, but in others we call `RestClient` directly?  This is another one we don't necessarily have to fix on this PR, but it struck me as inconsistent.